vsphere/tests/test_unit.py

```diff
@@ -3599,6 +3599,39 @@ def test_vsan_cluster_with_host_filtered(realtime_instance, dd_run_check, caplog
     assert "Skipping host world because it was filtered" in caplog.text
 
 
+@pytest.mark.usefixtures("mock_type", "mock_threadpool", "mock_api", "mock_rest_api")
+def test_vsan_cluster_to_redapl(realtime_instance, dd_run_check, datadog_agent, caplog):
+    realtime_instance['collect_vsan_data'] = True
+    caplog.set_level(logging.DEBUG)
+    check = VSphereCheck('vsphere', {}, [realtime_instance])
+    with patch('datadog_checks.vsphere.VSphereCheck.query_vsan_metrics') as MockQueryVsanMetrics:
```

Comment:
Could we get full test coverage for this feature?

Reply:
Talked over slack about this.